web_hide_db_manager_link/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Hide link to database manager in login screen
+=============================================
+
+This module hides the "Manage Databases" link at the bottom of login screen.
+
+The image in *web_hide_db_manager_link/static/src/img/screen.png* shows the resulting loging screen.
+
+.. image:: web_hide_db_manager_link/static/src/img/screenshot.png
+
+Installation
+============
+
+To install this module, you need to:
+
+ * Go to Settings / Local Modules
+ * Search by module name "Hide link to database manager in login screen" or by module technical name "*web_hide_db_manager_link*" 
+ * Click install button
+ 
+In case this module is not shown in your module list by default, click on "Update Module List" and clear any filter to show all modules.
```

Comment:
@legalsylvain This is the line already fixing your previous message in the other file. Is this OK?